perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ExtSourcesManagerBlImpl.java

```diff
@@ -259,23 +314,15 @@ public Candidate getCandidate(PerunSession perunSession, Map<String,String> subj
 			candidate.setServiceUser(false);
 		} else {
 			String isServiceUser = subjectData.get("isServiceUser");
-			if(isServiceUser.equals("true")) {
-				candidate.setServiceUser(true);
-			} else {
-				candidate.setServiceUser(false);
-			}
+			candidate.setServiceUser(isServiceUser.equals("true"));
 		}
 
 		//Set sponsored user
 		if(subjectData.get("isSponsoredUser") == null) {
 			candidate.setSponsoredUser(false);
 		} else {
 			String isSponsoredUser = subjectData.get("isSponsoredUser");
-			if(isSponsoredUser.equals("true")) {
-				candidate.setSponsoredUser(true);
-			} else {
-				candidate.setSponsoredUser(false);
-			}
+			candidate.setSponsoredUser(isSponsoredUser.equals("true"));
```

Comment:
These two changes are not part of the logic of this PR. If you insist to change them here, you need to add them to the commit message.